src/v/kafka/server/group_manager.cc

```diff
@@ -307,6 +307,15 @@ ss::future<> group_manager::recover_partition(
         }
     }
 
+    for (auto& [group_id, group_stm] : ctx.groups) {
+        auto group = get_group(group_id);
```

Comment:
this will often get confused with raft groups. it's a nit, but would help readability (since `cluster` namespace already picked `group` if the method does a `get_consumer_group(group_id)` vs `get_group()` which unless one looks at the namespace, it could be either a raft or a consumer group.

Reply:
`get_group` was there before this PR, so renaming it is out of scope, created [an issue](https://github.com/vectorizedio/redpanda/issues/1205) to address it later